research/gaia-x/pegasus/server/new_proto/new_job_service.proto

```diff
@@ -0,0 +1,139 @@
+/**
```

Comment:
BTW, please use proto/new_job_service.proto . In addition, add a TODO marker to show that this will replace job_service.proto if .... , kind of like that